examples/machine-learning/a3-highgpu-8g/a3high-slurm-blueprint.yaml

```diff
@@ -0,0 +1,711 @@
+            hosts: all
+            become: true
+            tasks:
+            - name: Find all standard GCP kernel packages
+              ansible.builtin.shell:
+                cmd: |
+                  dpkg -l | grep -E 'linux-(image|headers)-[0-9].*-gcp' | grep -v tcpx | awk '{print $2}'
+              register: gcp_kernel_packages
+              changed_when: false
+              check_mode: no # Always run this check, even in check mode
+            - name: Remove all standard GCP kernel packages
+              ansible.builtin.package:
+                name: "{{ ['linux-gcp', 'linux-image-gcp', 'linux-headers-gcp'] + gcp_kernel_packages.stdout_lines }}"
+                state: absent
+                autoremove: yes
+              when: gcp_kernel_packages.stdout_lines | length > 0
+              notify: Update GRUB
+            handlers:
+              - name: Update GRUB
+                ansible.builtin.command: update-grub
+
+- group: image
+  modules:
+  - id: slurm-image
+    source: modules/packer/custom-image
+    kind: packer
+    use:
+    - image_build_script
+    - sysnet
+    settings:
+      # building this image does not require a GPU-enabled VM but must *not* be
+      # run on a N-series VM otherwise, the "open" drivers will not install
+      machine_type: c2d-standard-32
+      source_image_project_id: [$(vars.source_image_project_id)]
+      source_image_family: $(vars.source_image_family)
+      image_family: $(vars.instance_image.family)
+      disk_size: $(vars.disk_size_gb)
+      omit_external_ip: false
+
+- group: cluster
+  modules:
+  - id: compute_sa
+    source: community/modules/project/service-account
+    settings:
+      name: compute
+      project_roles:
+      - logging.logWriter
+      - monitoring.metricWriter
+      - storage.objectAdmin
+
+  - id: a3_startup
+    source: github.com/GoogleCloudPlatform/cluster-toolkit//modules/scripts/startup-script?ref=v1.51.1&depth=1
+    settings:
+      # When shutting down a VM with local SSD disks, we strongly recommend the
+      # automatic migration of data following these instructions:
+      # https://cloud.google.com/compute/docs/disks/local-ssd#stop_instance
+      # Failure to do will result in VMs that lose data and do not automatically
+      # mount local SSD filesystems
+      local_ssd_filesystem:
+        mountpoint: $(vars.localssd_mountpoint)
+        permissions: "1777" # must quote numeric filesystem permissions!
+      # Docker was successfully installed in the image, this configures it
+      # to use the A3-specific local SSD volumes to store container images
+      docker:
+        enabled: true
+        world_writable: true
+        daemon_config: |
+          {
+            "data-root": "$(vars.localssd_mountpoint)/docker"
+          }
+      runners:
+      - type: ansible-local
+        destination: enable_nvidia_dcgm.yml
+        content: |
+          ---
+          - name: Enable NVIDIA DCGM on GPU nodes
```

Comment:
we really should automate this as part of the boot of images so it doesn't have to be here...  (not in this PR).